payments-core/src/main/java/com/stripe/android/model/PaymentIntent.kt

```diff
@@ -154,6 +154,10 @@ constructor(
 
     private val paymentMethodOptionsJsonString: String? = null,
 
+    val requireCvcRecollection: Boolean = paymentMethodOptionsJsonString?.let { json ->
+        JSONObject(json).optJSONObject("card")?.optBoolean("require_cvc_recollection") ?: false
+    } ?: false
+
```

Comment:
Do we need to make this a `constructor` property on `PaymentIntent`? Can we instead make it attribute on `PaymentIntent`?

Reply:
Nit: Can we make `card` and `require_cvc_recollection constants?

Reply:
I added it to the constructor so it would be parcelized, otherwise I can write custom parcelization logic